src/main/java/digital/slovensko/autogram/core/Autogram.java

```diff
@@ -56,22 +57,37 @@ public void checkPDFACompliance(SigningJob job) {
             return;
 
         ui.onWorkThreadDo(() -> {
-            var result = new PDFAStructureValidator().validate(job.getDocument());
+            // PDF/A doesn't support encryption
+            if (job.getDocument().hasOpenDocumentPassword()) {
+                ui.onUIThreadDo(() -> ui.onPDFAComplianceCheckFailed(job));
```

Comment:
ešte uvažujem, či sa neplatí vyrobiť pre tento prípad inú hlášku. Prípadne vedieť do toho `onPDFAComplianceCheckFailed` poslať ešte AutogramException, na základe ktorej sa zobrazí taký alebo onaký text. Totiž, pri tomto hesle má zmysel povedať userovi konkrétne, že nie len že dokument nie je v súlade s PDF/A, ale je to práva preto, že je zaheslovaný.

Reply:
Urcite ano. Poslime tam `ui.onPDFPasswordProtectedCheck(job)`